src/Harry/feature.js

```diff
@@ -1 +1,67 @@
-console.log("Harry");
\ No newline at end of file
+
+console.log("Harry");
+/*Activity one (data types)
+=========================
+What data type:
+
+14 ------------------ Number
+-80 ------------------ Number
+8.5 ---------------- Number
+true --------------- Boolean
+(5 < 7) ------------------ Boolean
+null ------------------ Undefined
+3.141 --------------- Number
+"Pi" ---------------- String
+67+18 --------------- Number 
+*/
+
+//Activity two//
+var convertToBinary = function(x){
+
+var result = [];
+while (x){
+	var remainder = x % 2;
+	result.push(remainder)
+x= (x/2) | 0;
+}
+result.reverse()
+return result.join("");
+}
+ 
+console.log(convertToBinary(10));
+
+
+//Activity three//
+
+countsB = function(x) {
```

Comment:
It's recommended that you name your parameters in such a way that their use can be inferred from the name.